src/Illuminate/Contracts/Auth/Guard.php

```diff
@@ -21,7 +21,7 @@ public function guest();
     /**
      * Get the currently authenticated user.
      *
-     * @return \Illuminate\Contracts\Auth\Authenticatable|null
+     * @return \Illuminate\Contracts\Auth\Authenticatable|\App\User|null
```

Comment:
You cant hardcode `\App\User` here.

Firstly - what happens if you have your User class elsewhere (I dont keep mine there).

Secondly - not everyone uses the `App` namespace.